graphql/schema/types/config.graphql

```diff
@@ -303,55 +303,95 @@ type TriggerConfig {
 input AdminSettingsInput {
   amboy: AmboyConfigInput
   amboyDB: AmboyDBConfigInput @redactSecrets
+  api: APIConfigInput
   authConfig: AuthConfigInput
   banner: String
   bannerTheme: BannerTheme
+  buckets: BucketsConfigInput
   cedar: CedarConfigInput
+  configDir: String
+  disabledGQLQueries: [String!]
+  domainName: String
+  expansions: StringMap
   fws: FWSConfigInput
+  githubCheckRun: GitHubCheckRunConfigInput
+  githubOrgs: [String!]
+  githubPRCreatorOrg: String
+  githubWebhookSecret: String @redactSecrets
+  hostInit: HostInitConfigInput
+  hostJasper: HostJasperConfigInput
   jira: JiraConfigInput
-  slack: SlackConfigInput
-  splunk: SplunkConfigInput
-  runtimeEnvironments: RuntimeEnvironmentConfigInput
-  testSelection: TestSelectionConfigInput
-  serviceFlags: ServiceFlagsInput
+  jiraNotifications: JiraNotificationsConfigInput
+  kanopySSHKeyPath: String @redactSecrets
+  logPath: String
+  loggerConfig: LoggerConfigInput
   notify: NotifyConfigInput
-  taskLimits: TaskLimitsConfigInput
-  hostInit: HostInitConfigInput
   podLifecycle: PodLifecycleConfigInput
-  scheduler: SchedulerConfigInput
+  pprofPort: String
+  projectCreation: ProjectCreationConfigInput
   repotracker: RepotrackerConfigInput
-  api: APIConfigInput
-  ui: UIConfigInput
-  disabledGQLQueries: [String!]
-  loggerConfig: LoggerConfigInput
+  runtimeEnvironments: RuntimeEnvironmentConfigInput
+  scheduler: SchedulerConfigInput
+  serviceFlags: ServiceFlagsInput
+  shutdownWaitSeconds: Int
+  singleTaskDistro: SingleTaskDistroConfigInput
+  slack: SlackConfigInput
+  sleepSchedule: SleepScheduleConfigInput
+  spawnhost: SpawnHostConfigInput
+  splunk: SplunkConfigInput
+  ssh: SSHConfigInput
+  taskLimits: TaskLimitsConfigInput
+  testSelection: TestSelectionConfigInput
+  tracer: TracerSettingsInput
   triggers: TriggerConfigInput
+  ui: UIConfigInput
 }
```

Comment:
I suspect that we're missing fields from `evergreen.Settings`; namely I see we don't have `PerfMonitoringURL`, `PerfMonitoringKanopyURL`, `ShutdownWaitSeconds`, `ReleaseMode`. Do you want to add them here?